res/controllers/Denon-MC7000-scripts.js

```diff
@@ -888,6 +894,41 @@ MC7000.crossFaderCurve = function(control, value) {
     script.crossfaderCurve(value);
 };
 
+// Update state on deck changes
+MC7000.switchDeck = function(channel, control, value, status, group) {
+    var deckOffset = script.deckFromGroup(group) - 1;
```

Comment:
```suggestion
    var deckOffset = control - 0x90;
```
Potentially faster as `script.deckFromGroup` does some expensive string and regex stuff to extract the decknumber

Reply:
I think we could even use `channel - 1`, but since the rest of the script uses `deckFromGroup` I went this that first.